test/models/prx_importer_test.rb

```diff
@@ -86,9 +86,17 @@ def pi.whitelisted?(aid)
         stub_request(:get, "https://cms.prx.org/api/v1/").
           to_return(:status => 200, :body => json_file(:prx_root), :headers => {})
 
+        # test create, should be in the cms
         stub_request(:get, "https://cms.prx.org/api/v1/stories/87683").
           to_return(:status => 200, :body => json_file(:prx_story), :headers => {})
 
+        # test delete, should not be there
+        stub_request(:get, "https://cms.prx.org/api/v1/stories/20000").
+          to_return(:status => 404, :body => '{}', :headers => {})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.